tests/clients/test_pika.py

```diff
@@ -78,10 +80,12 @@ def test_basic_publish(self, send_method, _unused):
             pika.spec.Basic.Publish(
                 exchange="test.exchange",
                 routing_key="test.queue"), (pika.spec.BasicProperties(headers={
-                    "X-INSTANA-T": rabbitmq_span.t,
-                    "X-INSTANA-S": rabbitmq_span.s,
-                    "X-INSTANA-L": "1"
-                }), b"Hello!"))
+                'traceparent': '00-0000000000000000{}-{}-01'.format(rabbitmq_span.t, rabbitmq_span.s),
```

Comment:
[AMPQ](https://w3c.github.io/trace-context-amqp/) has a different format used to propagate the headers.

Reply:
I got an idea on fully decoupling the propagators which need trace context parsing and propagating versus the ones not and I am gonna give it a try